src/feature/api/channels/domain/channel-message.ts

```diff
@@ -0,0 +1,35 @@
+export class ChannelMessage {
+	private readonly _id: string;
+	private readonly _participantId: string;
+	private readonly _channelId: string;
+	private readonly _content: string;
+
+  // TODO: implement validation
+	constructor(props: {
+		_id: string;
+		_participantId: string;
+		_channelId: string;
+		_content: string;
```

Comment:
```suggestion
		content: string;
```
클래스 내부에서 사용하는것만 프리픽스로 _붙입시다